app/public/views/flags/Flags.js

```diff
@@ -38,7 +38,12 @@ export default class Flags extends Observable {
 
     getFlags(runNumber, detector) {
         const allFlags = this.getAllFlags();
-        const flags = Array.isArray(allFlags) ? allFlags : [];
+        const flags = !Array.isArray(allFlags) ? [] : allFlags.map((flag) => {
+            const { verifications } = flag;
+            flag.ver_time = verifications?.map(({ verification_time }) => verification_time);
```

Comment:
Avoid shortcuts such as `ver` for `verification`. Gain is null when using an IDE and cognitive cost is high.
Also, are those properties shortcut really useful? I think that `flag.verification.verification_time` is way more clear than `flag.ver_time`

Reply:
1. `ver` replaced by `verification`
2. I know it's strange, but for now it must be like that, because, changing it requires a lot of changes in table building methods which don't work with nested objects